src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorSingleStream.java

```diff
@@ -65,7 +64,7 @@
 
     private long lastProcessTime = -1L;
 
-    private final RemoteBitrateObserver observer;
+    private final VideoMediaStreamImpl stream;
```

Comment:
I would prefer to keep the code as close to the original as possible -- keep the observer as "observer", and have VideoMediaStreamImpl implement the interface. What do you think? We can still call hashCode on it, which seems like the only use so far (but perhaps you have something else in mind for the future).